content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,64 @@
 ---
 reviewers:
 - nelvadas
-title: Limit Ranges
+title: LimitRanges
 content_template: templates/concept
 weight: 10
 ---
 
 {{% capture overview %}}
 
-By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
-With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
-Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one Pod or Container could monopolize all of the resources. Limit Range is a policy to constrain resource  by Pod or Container in a namespace.
+By default, Containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
```

Comment:
I prefer the original line. The term “container” here is about the general IT concept rather than an element of Kubernetes' API.

Reply:
Sure, I will return to lowercase. My fault. I didn't realize capitals were reserved for API concepts.